cloudformation-templates/zs_cc_cf_template_simple.yaml

```diff
@@ -140,11 +146,18 @@ Parameters:
         AllowedValues:
           - true
           - false
-  VPCIpCidrBlock:
+  ServiceIntSGCidr:
```

Comment:
Any migration challenges for customers using the template right now? in the description should we mention that this was previously VPCIpCidrBlock?

Reply:
yeah sorry this PR is a bit confusing as i added a couple cidr/ip related parameters and then opted to remove them after further discussion